lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -221,6 +232,10 @@ export default Relay.createContainer(ArtworkRail, {
       fragment on HomePageArtworkModule {
         ${Header.getFragment('rail')}
         key
+        params {
+          medium
+          price_range
+        }
```

Comment:
didn't really think I could put that in a genericGene fragment, but maybe I could?